Lambda-School-Labs/labs9-stock-trainer
# Topbar

Description
This adds a top bar with logic for logged in and out states. 

Fixes # (issue)

## Type of change


# How Has This Been Tested?

1. Ensure you have yarn installed on your system
2. Run `yarn install`
3. Run `yarn start`
4. Navigate to `localhost:3000`

# Checklist:
